app/dts/behaviors/ext_power.dtsi

```diff
@@ -7,7 +7,7 @@
 / {
     behaviors {
         // Behavior can be invoked on peripherals, so name must be <= 8 characters.
-        ext_power: extpower {
+        /omit-if-no-ref/ ext_power: extpower {
```

Comment:
This was intentional, so probably needs an added comment, instead of adding the `omit-if-no-ref`.